### PR TITLE
Added refund option

### PR DESCRIPTION
**Pull request by abelpuentes**
Can you update the library on pypi too?

Thanks!

### CODE REVIEW

File: sofort/model.py [hunk 1]
```diff
@@ -116,7 +116,26 @@ class TransactionDetailsModel(Model):
     costs = ModelType(CostsModel)
     su = ModelType(SuModel)
     status_history_items = SofortListType('status_history_item',
-                                          ModelType(StatusHistoryItemModel))
+                                          ModelType(StatusHistoryItemModel)))
```
Comment (devsli) [inline sofort/model.py]: Travis CI said that you have `SyntaxError` here

Comment (abelpuentes) [reply sofort/model.py]: Fixed, thanks!

**Comment by abelpuentes** [comment]
Could you please update the library on pypi? Thanks!

**Comment by devsli** [comment]
Sorry, I can't accept this pull request until tests are passed or we decide that tests are wrong.
No idea why it keeps failing.

**Comment by abelpuentes** [comment]
Fixed the issue and improved coverage. Thanks!

**Comment by abelpuentes** [comment]
Did you check my pull-request? Thanks!

**Comment by devsli** [comment]
Thanks a lot! Uploaded new version to the PyPI

https://pypi.python.org/pypi/sofort/0.4.3

**Pull request merged**